### ISSUE TITLE
wmbus

### ISSUE THREAD

**Issue by charugaa** [things4u/ESP-1ch-Gateway#44]
can fsk 868.95 be received with your gateway ... european wmbus

**Comment by platenspeler** [comment]
No, I'm sorry but FSK has not been tested and will not work (as gar as I know)
